### PR TITLE
[MIG] account_invoicing_mode*: Migration to 14.0

### PR DESCRIPTION
**Pull request by simahawk**
Oca port pr 755 from 13.0 to 14.0. Replaces  #844, rebased on latest 13.0

### CODE REVIEW

File: account_invoice_mode_at_shipping/models/stock_picking.py [hunk 1]
```diff
@@ -3,29 +3,25 @@
...
+    def _action_done(self):
+        res = super()._action_done()
         for picking in self:
             if picking._invoice_at_shipping():
                 picking.with_delay()._invoicing_at_shipping()
         return res
 
     def _invoice_at_shipping(self):
-        """Check if picking must be invoiced at shipping."""
+        """Check if picking must be invoiced at shippin`g."""
```
Comment (mmequignon) [inline account_invoice_mode_at_shipping/models/stock_picking.py]: shipping

**Review by mmequignon — APPROVED** [review]
LG, just a nitpick

**Review by SilvioC2C — APPROVED** [review]
LGTM

**Comment by simahawk** [comment]
```
2021-08-03 14:50:20,988 6091 ERROR openerp_test odoo.addons.account_invoice_date_due.tests.test_account_invoice_date_due: ERROR: TestAccountInvoiceDateDue.test_invoice_date_due_is_editable_w_payment_term

Traceback (most recent call last):

File "/home/travis/odoo-14.0/odoo/tools/cache.py", line 85, in lookup

r = d[key]

File "/home/travis/odoo-14.0/odoo/tools/func.py", line 71, in wrapper

return func(self, *args, **kwargs)

File "/home/travis/odoo-14.0/odoo/tools/lru.py", line 34, in __getitem__

a = self.d[obj]

KeyError: ('ir.model.access', <function IrModelAccess.check at 0x7f87a26032f0>, 10, False, 'stock.valuation.layer', 'read', True, (None,))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

File "/home/travis/build/OCA/account-invoicing/setup/account_invoice_date_due/odoo/addons/account_invoice_date_due/tests/test_account_invoice_date_due.py", line 135, in test_invoice_date_due_is_editable_w_payment_term

old_move_state = self.move.read()[0]

File "/home/travis/odoo-14.0/odoo/models.py", line 3014, in read

self._read(stored_fields)

File "/home/travis/odoo-14.0/odoo/models.py", line 3160, in _read

field.read(fetched)

File "/home/travis/odoo-14.0/odoo/fields.py", line 3158, in read

lines = comodel.search(domain, limit=self.limit)

File "/home/travis/odoo-14.0/odoo/models.py", line 1702, in search

res = self._search(args, offset=offset, limit=limit, order=order, count=count)

File "/home/travis/odoo-14.0/odoo/models.py", line 4481, in _search

model.check_access_rights('read')

File "/home/travis/odoo-14.0/odoo/models.py", line 3327, in check_access_rights

return self.env['ir.model.access'].check(self._name, operation, raise_exception)

File "<decorator-gen-34>", line 2, in check

File "/home/travis/odoo-14.0/odoo/tools/cache.py", line 90, in lookup

value = d[key] = self.method(*args, **kwargs)

File "/home/travis/odoo-14.0/odoo/addons/base/models/ir_model.py", line 1805, in check

raise AccessError(msg)

odoo.exceptions.AccessError: You are not allowed to access 'Stock Valuation Layer' (stock.valuation.layer) records.

This operation is allowed for the following groups:

- Inventory/Administrator

Contact your administrator to request access if necessary.
```
To be checked why

**Pull request merged**